app/connectors/sharedmodel/CalculatorResult.scala

```diff
@@ -33,7 +34,7 @@ case class MarginalRate(
   adjustedLowerThreshold: Double,
   adjustedUpperThreshold: Double
 ) extends TaxDetails {
-  def adjustedAugmentedProfit: Double = adjustedProfit + adjustedDistributions
+  def adjustedAugmentedProfit: Double = roundUp(BigDecimal(adjustedProfit) + BigDecimal(adjustedDistributions))
```

Comment:
Need to roundup, as decimal additional may endup with more than 2 decimal places